src/common/utils/NebulaKeyUtils.h

```diff
@@ -251,6 +258,108 @@ class NebulaKeyUtils final {
         return rawKey.subpiece(0, rawKey.size() - sizeof(int64_t));
     }
 
+    // Only int and double are supported
+    static std::string boundVariant(nebula::cpp2::SupportedType type,
+                                    NebulaBoundValueType op,
+                                    const VariantType& v = 0L) {
+        switch (op) {
+            case NebulaBoundValueType::kMax : {
+                std::vector<unsigned char> bytes = {0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF};
```

Comment:
Is it right for int64_t and double?

Reply:
> Is it right for int64_t and double?

int64 and double convert to binary as below, I think using byte of filled with '0xFF'  is ok. WDYT?
std::numeric_limits<int64_t>::max() -->0xfffffffffffff
std::numeric_limits<double>::max() --> -nan(0xfffffffffefff)

Reply:
because here also handle double type. so to do it as std::vector<unsigned char> bytes